install.sh

```diff
@@ -0,0 +1,98 @@
+#/bin/bash
+# The script requires root permissions
+
+function install_trento() {
+    . /etc/os-release
+    if [[ ! $NAME =~ "SUSE" ]]; then
+        echo "Not SUSE operating system."
+        return -1
+    fi
+    case "$VERSION_ID" in
+    "15.3")
+        repo="https://download.opensuse.org/repositories/devel:/sap:/trento/15.3/devel:sap:trento.repo"
+        ;;
+    *)
+        echo "Not supported OS release."
```

Comment:
This sounds a bit weird to me, I'd write "Unsupported OS release" or "OS release not supported"

Reply:
Changed to "Operating system is not supported."